torch/nn/functional.py

```diff
@@ -4716,15 +4716,17 @@ def _in_projection_packed(
     if k is v:
         if q is k:
             # self-attention
-            return linear(q, w, b).chunk(3, dim=-1)
```

Comment:
@ansley per offline chat, it'd be better to fix your test case by using `# type: ignore[return-value]` here - since `chunk()` actually does return a tuple, the new code on line 4720 is literally just building a second tuple from the members of the first. 

Similarly on line 4727, though you might need `# type: ignore[operator, return-value]` there. (LMK if this doesn't fix the errors you were seeing - we might need to do something more, but one way or another I'm sure there's a way to quiet mypy without doing the gratuitous tuple-rebuilding at runtime.)

Also though, I'd suggest breaking this out to a separate PR, since this isn't really thematically related to the rest of what you're up to here.

cc @gmagogsfm - background here is that `torch.chunk()` has a schema return type of `Tensor[]`, but actually returns a Python `Tuple[Tensor, ...]` in eager.

Reply:
Fixed. See Basil's comment